package.json

```diff
@@ -9,14 +9,14 @@
     "type": "git"
   },
   "author": "Mapbox",
-  "version": "3.3.0",
+  "version": "3.3.1-dev2",
```

Comment:
I will uptick the version on merge. Could you please revert this (you can leave the changelog changes).

Reply:
```suggestion
  "version": "3.3.0",
```